<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>666</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>992</width>
    <height>666</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>992</width>
    <height>666</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">groove:horizontal { background-color: gray; }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="GLWidget" name="GLwidget" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">groove:horizontal { background-color: gray; }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_obj">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>560</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Open object</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Смещение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="move_x_Box">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>-20.000000000000000</double>
    </property>
    <property name="maximum">
     <double>20.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="move_y_Box">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-20.000000000000000</double>
    </property>
    <property name="maximum">
     <double>20.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="move_z_Box">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-20.000000000000000</double>
    </property>
    <property name="maximum">
     <double>20.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rotation_y_Box">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>-180.000000000000000</double>
    </property>
    <property name="maximum">
     <double>180.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rotation_z_Box">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>-180.000000000000000</double>
    </property>
    <property name="maximum">
     <double>180.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вращение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rotation_x_Box">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>-180.000000000000000</double>
    </property>
    <property name="maximum">
     <double>180.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Размер</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scale_obj_Box">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>-10.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Масшабирование</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Название:</string>
    </property>
   </widget>
   <widget class="QLabel" name="count_vertex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Кол-во вершин:</string>
    </property>
   </widget>
   <widget class="QLabel" name="count_facets">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Кол-во ребер:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>992</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
